GitHubSearch/Services/RepositoryService.swift

```diff
@@ -33,12 +33,14 @@ final class RepositoryService {
     }
 
     func toggleFavorite(_ repository: Repository) {
-        repository.isFavorite.toggle()
+        realmService?.update {
+            repository.isFavorite = !repository.isFavorite
+        }
+        
         if repository.isFavorite {
-            realmService?.save(repository)
+            self.realmService?.save(repository)
```

Comment:
Why do you use `self`? It's not a closure.

Reply:
You can delete it later, sorry i didn't notice that.

Reply:
Can you do it yourself?

Reply:
ok i will make another pr